.ci_support/migrations/remove-python38.yaml

```diff
@@ -0,0 +1,26 @@
+__migrator:
```

Comment:
This is a trick that we can use to remove a particular variant key from the expanded set